aw-datastore/src/legacy_import.rs

```diff
@@ -139,7 +116,35 @@ mod import {
         let mut list = Vec::new();
         for row in rows {
             match row {
-                Ok(event) => list.push(event),
+                Ok((timestamp_str, duration_float, data_str)) => {
+                    let timestamp_str = timestamp_str.replace(" ", "T");
+                    let timestamp = match DateTime::parse_from_rfc3339(&timestamp_str) {
+                        Ok(timestamp) => timestamp.with_timezone(&Utc),
+                        Err(err) => panic!("Timestamp string {}: {:?}", timestamp_str, err),
+                    };
+
+                    let duration_ns = (duration_float * 1_000_000_000.0) as i64;
+
+                    let data: serde_json::map::Map<String, serde_json::Value> =
+                        match serde_json::from_str(&data_str) {
+                            Ok(data) => data,
+                            Err(err) => {
+                                warn!(
+                                    "Unable to parse JSON data in event from bucket {}\n{}\n{}",
+                                    bucket_id, err, data_str
+                                );
+                                continue;
```

Comment:
I'd prefer if the invalid string in question was replaced with "invalid" or something instead of just skipped with a warning, but no biggie.